nodejs/awsx/apigateway/api.ts

```diff
@@ -118,17 +118,51 @@ function isStaticRoute(route: Route): route is StaticRoute {
 }
 
 /**
- * An apigateway route that maps to some target uri, or some elastic-load-balancer host/port.
+ * An apigateway route for an integration.
+ * https://docs.aws.amazon.com/apigateway/api-reference/resource/integration/ for more details.
  */
-export interface ProxyRoute {
+export interface IntegrationRoute {
```

Comment:
note: this is technically a breaking change. however, awsx-apigateway is so new i think it can be acceptable for us to do this.  Alternativly, we could keep the 'proxy' name, though 'integration' really is better given what this represents.

What do you think @lukehoban